lib/warden/action_dispatch_patch.rb

```diff
@@ -0,0 +1,24 @@
+# ActionDispatch has an authorization method but Rack::Request does not.
```

Comment:
nah

On Tue, Aug 9, 2016 at 11:32 AM, Adam Panzer <notifications@github.com>
wrote:

> In lib/warden/action_dispatch_patch.rb
> <https://github.com/zendesk/samson/pull/1163#discussion_r74117814>:
>
> > @@ -0,0 +1,24 @@
> > +# ActionDispatch has an authorization method but Rack::Request does not.
>
> ehhhh. Very low value in fixing this, do you this change is imperative?
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub
> <https://github.com/zendesk/samson/pull/1163/files/ec9176e57ff6e9773ea877bdef9761f323850b46#r74117814>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/AAAsZyh2sJ_sf33wc1NZJpQ6FcVgnzjmks5qeMehgaJpZM4JVi_g>
> .
>